packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -53,27 +53,28 @@ class FlutterDevice {
     this.userIdentifier,
     required this.developmentShaderCompiler,
     this.developmentSceneImporter,
-  }) : generator = generator ?? ResidentCompiler(
-         globals.artifacts!.getArtifactPath(
-           Artifact.flutterPatchedSdkPath,
-           platform: targetPlatform,
-           mode: buildInfo.mode,
-         ),
-         buildMode: buildInfo.mode,
-         trackWidgetCreation: buildInfo.trackWidgetCreation,
-         fileSystemRoots: buildInfo.fileSystemRoots,
-         fileSystemScheme: buildInfo.fileSystemScheme,
-         targetModel: targetModel,
-         dartDefines: buildInfo.dartDefines,
-         packagesPath: buildInfo.packageConfigPath,
-         frontendServerStarterPath: buildInfo.frontendServerStarterPath,
-         extraFrontEndOptions: buildInfo.extraFrontEndOptions,
-         artifacts: globals.artifacts!,
-         processManager: globals.processManager,
-         logger: globals.logger,
-         platform: globals.platform,
-         fileSystem: globals.fs,
-       );
+  }) : generator = generator ??
+            ResidentCompiler(
+              globals.artifacts!.getArtifactPath(
+                Artifact.flutterPatchedSdkPath,
+                platform: targetPlatform,
+                mode: buildInfo.mode,
+              ),
+              buildMode: buildInfo.mode,
+              trackWidgetCreation: buildInfo.trackWidgetCreation,
+              fileSystemRoots: buildInfo.fileSystemRoots,
+              fileSystemScheme: buildInfo.fileSystemScheme,
+              targetModel: targetModel,
+              dartDefines: buildInfo.dartDefines,
+              packagesPath: buildInfo.packageConfigPath,
+              frontendServerStarterPath: buildInfo.frontendServerStarterPath,
+              extraFrontEndOptions: buildInfo.extraFrontEndOptions,
+              artifacts: globals.artifacts!,
+              processManager: globals.processManager,
+              logger: globals.logger,
+              platform: globals.platform,
+              fileSystem: globals.fs,
+            );
```

Comment:
Looks like this PR contains a lot of unintentional, automatically-done formatting changes that blow up the diff and violate some of our current style guide. As one class of examples, look for long declare-and-assign statements that got broken up into two lines.

I know we are going to be running `dart format` over everything soon enough anyway, but I would like all that noise to be bundled in its own PR for the sake of folks running `git blame` in the future.

Reply:
I unformatted.